### DISCUSSION TITLE
Using code for my website

### DISCUSSION THREAD

**Discussion by toa-23** [dayoonkwon/BioAge#39] [Q&A]
@dayoonkwon 

Hi, 
Can I use the code available here: https://github.com/dayoonkwon/BioAge/ to offer a free Biological age testing tool on my website? 

I would include disclaimers that state the output is for informational purposes only and should not be taken as a replacement for professional medical advice. 

I appreciate your response. 

Thanks

**Comment by dayoonkwon** [comment]
Thank you for reaching out regarding the use of my code. 

I suggest the following: 1) Make sure to acknowledge me and the paper as the source of the code. 2) Ensure that the disclaimer is clear and that you are not selling anything on your website, or at least on the page that includes the BioAge calculator.
